tempesta_fw/http_parser.c

```diff
@@ -126,14 +136,58 @@ do {									\
  * header values. That is done with separate, nested, or interior
  * FSMs, and so _I_ in the name means "interior" FSM.
  */
+
 #define __FSM_I_EXIT()			goto done
-#define __FSM_I_MOVE_n(to, n)						\
+#define __FSM_I_CHUNK_FIXUP(n)						\
+do {									\
+	/* Save symbols until current, plus n symbols more */	\
+	int slen = p - data + n;					\
+	__chunk_finish(msg, data, slen, &__fsm_chunkflags);		\
```

Comment:
```__fsm_chunkflags``` is defined in only one place, so the macros using the variable are also used in only one place, so it's better to remove them and do the code w/o macros.